File: src/main/java/lightninglobby/lightninglobby/stopdrop.java
package lightninglobby.lightninglobby;

import org.bukkit.entity.Player;
import org.bukkit.event.EventHandler;
import org.bukkit.event.Listener;
import org.bukkit.event.block.BlockDropItemEvent;
import org.bukkit.event.player.PlayerDropItemEvent;

public class stopdrop implements Listener {
    @EventHandler
    public void BlockDropItemEvent(PlayerDropItemEvent event){
        if (event.getPlayer().getLocation().getWorld().getName().equals("Minigames_Lobby"))
            event.setCancelled(true);
    }
}

File: src/main/kotlin/lightninglobby.lightninglobby/StopDrop.kt
package lightninglobby.lightninglobby

import org.bukkit.event.EventHandler
import org.bukkit.event.Listener
import org.bukkit.event.player.PlayerDropItemEvent

class StopDrop : Listener {
    @EventHandler
    fun blockDropItemEvent(event: PlayerDropItemEvent) {
        if (event.player.location.world.name == "Minigames_Lobby") event.isCancelled = true
    }
}